src/Tools/Source/RunTests/TestRunner.cs

```diff
@@ -127,7 +127,7 @@ internal async Task<RunAllResult> RunAllOnHelixAsync(ImmutableArray<WorkItemInfo
                 executable: _options.DotnetFilePath,
                 arguments: "build helix-tmp.csproj",
                 captureOutput: true,
-                onOutputDataReceived: (e) => ConsoleUtil.WriteLine(e.Data),
+                onOutputDataReceived: (e) => { Debug.Assert(e.Data is not null); ConsoleUtil.WriteLine(e.Data); },
```

Comment:
Getting `##[error]src\Tools\Source\RunTests\TestRunner.cs(130,68): error CS8604: (NETCORE_ENGINEERING_TELEMETRY=Build) Possible null reference argument for parameter 'message' in 'void ConsoleUtil.WriteLine(string message)'.` in CI correctness leg.

If you look at the implementation of `CreateProcess`, the `onOutputDataReceived` callback is only invoked when `e.Data` isn't `null`.